### PR TITLE
Added home page intro cards for logged in and anonymous users

### LINKED ISSUE #1242: Intro card for home page
The home page template is slightly different from channel pages. 

### acceptance criteria
- [ ] add static intro card
- [ ] for anonymous users, the intro card welcomes them and asks them to create an account. See https://zpl.io/VqE9Xqm
- [ ] for logged in users, the intro card welcomes them back. See https://zpl.io/2EAZJpY

### PR DESCRIPTION
**Pull request by gsidebo**
#### Pre-Flight checklist
- [x] Testing
  - [x] Code is tested
  - [x] Changes have been manually tested

#### What are the relevant tickets?
Closes #1242

#### What's this PR do?
Adds an intro card to the home page

#### How should this be manually tested?
Check the home page while logged in and not logged in

#### Any background context you want to provide?
Not-logged-in design: https://app.zeplin.io/project/5b181d3bf71408ae4d406b94/screen/5b60e6907ed98b73a5440778
Logged-in design:
https://app.zeplin.io/project/5b181d3bf71408ae4d406b94/screen/5b60e68e8e733e22302777bf

#### Screenshots (if appropriate)
ss 2018-09-26 at 15 14 57
ss 2018-09-26 at 15 20 18
ss 2018-09-26 at 15 20 32

ss 2018-09-26 at 13 45 42
ss 2018-09-26 at 13 36 38

### CODE REVIEW

**Comment by aliceriot** [comment]
currently the 'Become a member' link sends you to `/login`, should it send you to `/signup` instead?

**Comment by gsidebo** [comment]
homer_disappear.gif

**Comment by gsidebo** [comment]
@aliceriot fixed

**Comment by aliceriot** [comment]
one question for @pdpinch, do we want to merge this with the new branding (the Open logo)?

File: static/js/containers/HomePage.js [hunk 1]
```diff
@@ -77,6 +81,39 @@ export class HomePage extends React.Component<Props> {
...
+
+    return (
+      <Card className="home-callout">
+        <div className="logo-col">
+          <img src="/static/images/MIT_circle.svg" alt="MIT Logo" />
+        </div>
+        <div className="text-col">
+          <h3>{headerText}</h3>
+          <p>
+            A place where learners, teachers and scientist worldwide meet with
```
Comment (aliceriot) [inline static/js/containers/HomePage.js]: I think `scientist` should probably be `scientists` here

**Comment by pdpinch** [comment]
Oh, good question. We need the logo to be conditional. Only use the circle logo if `USE_NEW_BRANDING` is True. Otherwise use the current mit-logo-transparent3.svg logo

**Comment by gsidebo** [comment]
@pdpinch This is how it looks with old branding now. Let me know if you want anything changed
ss 2018-09-28 at 11 44 40

**Comment by gsidebo** [comment]
@aliceriot ready for review again

**Comment by pdpinch** [comment]
How is the size of the logo being determined? I feel like it should be the same height as the headerText plus the paragraph that follows.

**Comment by aliceriot** [comment]
@pdpinch I think this is waiting on some feedback

**Comment by gsidebo** [comment]
Updated screenshots based on latest design:

ss 2018-10-05 at 11 59 09
ss 2018-10-05 at 11 58 32
ss 2018-10-05 at 11 59 01
ss 2018-10-05 at 11 58 20
ss 2018-10-05 at 12 01 18

**Comment by pdpinch** [comment]
Thanks for the updated screenshots. This looks great. 

for the reviewer: we decided to use the same text, whether the user is logged in or not. Only the button changes.

**Comment by gsidebo** [comment]
(@aliceriot ready again)

**Pull request merged**